core/src/test/scala/unit/kafka/admin/ReassignPartitionsClusterTest.scala

```diff
@@ -478,7 +476,7 @@ class ReassignPartitionsClusterTest extends ZooKeeperTestHarness with Logging {
    */
   @Test
   def shouldPerformMultipleReassignmentOperationsOverVariousTopics() {
-    servers = TestUtils.createBrokerConfigs(4, zkConnect, false).map(conf => TestUtils.createServer(KafkaConfig.fromProps(conf)))
+    startBrokers(Seq(0, 1, 2, 3))
```

Comment:
startBrokers() enables controlled shutdown, which will slow down the shutting down of all brokers. Perhaps we should disable controlled shutdown in startBrokers()?

Reply:
Done.